packages/wallets/src/features/cfd/modals/DxtradeEnterPasswordModal/DxtradeEnterPasswordModal.tsx

```diff
@@ -24,15 +27,35 @@ const DxtradeEnterPasswordModal = () => {
         });
     };
 
+    const successDescription =
```

Comment:
better to add title and description for demo and real
```suggestion
    const successContentMapper ={
       demo: {
           titlle: 'Your Deriv X demo account is ready',
           description: 'Transfer virtual funds from your Demo Wallet to your Deriv X Demo account 
            to practise trading.'
       },
       real: {
             title: ...,
             description: ...
       }
    }
```

Reply:
We will move this out in the next PR me and @adrienne-deriv planning to do. We planning to cleanup cfd a bit by having a centralised type and config